book/tutorials/snowex_database/5_plot_raster_example.ipynb

```diff
@@ -0,0 +1,335 @@
+   "source": [
+    "# Forming Queries with Rasters\n",
+    "Querying the database with rasters is essentially the same as with the other two tables. A primary difference however is they are returned as a [rasterio dataset](https://rasterio.readthedocs.io/en/stable/api/rasterio.io.html#rasterio.io.DatasetReader) instead of a dataframe.\n",
+    "   \n",
+    "## Grab a whole raster. \n",
+    "Grabbing whole rasters can be done (albeit with caution) using the `from_filter` function. \n",
+    "\n",
+    "**Note**: snowexsql will throw an error if you try to pull more than one dataset at a time. This is because this function is merging tiles together based on the query and if the dataset grids dont match the database throws a cryptic error. So we took the liberty ahead of time. \n",
+    "\n",
+    "**Try this** : To see the error in action, remove the date from the query and run it. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'show' is not defined",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
+      "Cell \u001b[0;32mIn[2], line 12\u001b[0m\n\u001b[1;32m      9\u001b[0m ds \u001b[38;5;241m=\u001b[39m RasterMeasurements\u001b[38;5;241m.\u001b[39mfrom_filter(observers\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mASO Inc.\u001b[39m\u001b[38;5;124m'\u001b[39m, date\u001b[38;5;241m=\u001b[39mdt, \u001b[38;5;28mtype\u001b[39m\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mswe\u001b[39m\u001b[38;5;124m'\u001b[39m)\n\u001b[1;32m     11\u001b[0m \u001b[38;5;66;03m# Plot it up!\u001b[39;00m\n\u001b[0;32m---> 12\u001b[0m \u001b[43mshow\u001b[49m(ds[\u001b[38;5;241m0\u001b[39m], vmin\u001b[38;5;241m=\u001b[39m\u001b[38;5;241m0.1\u001b[39m, vmax\u001b[38;5;241m=\u001b[39m\u001b[38;5;241m0.4\u001b[39m, cmap\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mwinter\u001b[39m\u001b[38;5;124m'\u001b[39m)\n\u001b[1;32m     14\u001b[0m \u001b[38;5;66;03m# Note the resolution!\u001b[39;00m\n\u001b[1;32m     15\u001b[0m ds[\u001b[38;5;241m0\u001b[39m]\u001b[38;5;241m.\u001b[39mres\n",
+      "\u001b[0;31mNameError\u001b[0m: name 'show' is not defined"
+     ]
+    }
+   ],
+   "source": [
+    "# import in the raster measurments class\n",
+    "from snowexsql.api import RasterMeasurements\n",
+    "from datetime import datetime \n",
+    "\n",
+    "# Pick a date\n",
+    "dt = datetime(2020, 2, 13)\n",
+    "\n",
+    "# Query db filtering to swe on a certain date surveyed by ASO\n",
+    "ds = RasterMeasurements.from_filter(observers='ASO Inc.', date=dt, type='swe')\n",
+    "\n",
+    "# Plot it up!\n",
+    "show(ds[0], vmin=0.1, vmax=0.4, cmap='winter')\n",
```

Comment:
Executing this on cryocloud I'm getting a traceback: `NameError: name 'show' is not defined` . I think once you merge in the main branch and clear the output cells from these notebooks it'll be good to merge